pom.xml

```diff
@@ -81,6 +81,9 @@ The MIT License (MIT)
       <url>https://github.com/yegor256/cactoos</url>
     </site>
   </distributionManagement>
+  <prerequisites>
+    <maven>3.0.4</maven>
```

Comment:
this is not correct, prerequisites section are only for maven plugin projects, to define required maven version, we need to specify it via maven enforcer plugin in the build section

Reply:
Thanks, i've fixed it